libstorage/BinLogHandler.cpp

```diff
@@ -304,19 +304,39 @@ void BinLogHandler::encodeEntries(
                                   << LOG_KV("id", entry->getID())
                                   << LOG_KV("status", (uint32_t)status);*/
         buffer.insert(buffer.end(), (byte*)&status, (byte*)&status + 1);
+        int flagSize = (vecField.size() - 3) / 8 + 1;  // exclude STATUS,NUM_FIELD,ID_FIELD
+        uint8_t* usedFlag = new uint8_t[flagSize];     // bit-based used to represent a field
+        memset(usedFlag, 0, sizeof(uint8_t) * flagSize);
+        int KeyIdx = 0;  // field index
+        for (const auto& key : vecField)
+        {
+            if (key == STATUS || key == NUM_FIELD || key == ID_FIELD)
```

Comment:
```suggestion
            if (key == STATUS || key == NUM_FIELD || key == ID_FIELD)
```
This filiter can be done of vecField

Reply:
these three fields recorded are required when restoring table fields

Reply:
these three fields recorded are required when restoring table fields